resources/qml/SidebarHeader.qml

```diff
@@ -196,16 +196,63 @@ Column
             }
 
             anchors.verticalCenter: parent.verticalCenter
-            width: parent.width * 0.45 - UM.Theme.getSize("default_margin").width
+            width: parent.width * 0.30
             font: UM.Theme.getFont("default");
             color: UM.Theme.getColor("text");
         }
 
+        Text
+        {
+            width: parent.width * 0.05
+        }
+
+        Button
+        {
+            id: materialInfoButton
+            height: parent.height * 0.70
+            width: height
+            anchors.leftMargin: UM.Theme.getSize("default_margin").width
```

Comment:
For the next time you can just set all anchors with; anchors.margins: UM.Theme.getSize("default_margin").width